docs/user/analytics.rst

```diff
@@ -1,5 +1,8 @@
-Traffic Analytics
-=================
+How To Use Traffic Analytics
```

Comment:
Seems this is a How To guide, but not in the `guides` directory? 

Seems we should break this out into a How To for Google analytics, and the overall feature page which isn't a how to?

Reply:
Yes, this was on purpose though. I was thinking that the majority of the existing content (already index by Google) is a How-to. We have all the work with renaming files/folders and creating redirects postponed for later.

I agree that Google Analytics should be split into a separate guide. I'm going to ask some questions on Slack about its contents in this case.

Reply:
This should be noted in the file somewhere then, but I guess the title & being in a guide index is a reasonably good start.

Reply:
Yeah, I think we can keep it here for now. I fear that adding another page explaining analytics is a bit too much right now, because we have to tie together the SEO guide and existing contents that haven't yet trickled down into their correct navigation places.

Made a note of this in Iteration 4 :+1: